src/bci_build/package/php-apache/README.md.j2

```diff
@@ -0,0 +1,41 @@
+# The PHP Apache {{ image.version }} Container Image
+
+{% include 'badges.j2' %}
+
+{% include 'php_description.j2' %}
+
+## How to use this image
+
+This image ships with the Apache web server and the `mod_php` module.
+
+To deploy your application, copy your sources into the htdocs folder
+`/srv/www/htdocs` (this directory is the `WORKDIR` of the container image):
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+RUN set -eux; \
+    zypper -n in $my_dependencies; \
+    # additional setup steps
+
+# copy your app into the Apache2 document root
+COPY app/ .
+```
+
+Then build the image and run the resulting container:
+
+```ShellSession
+# buildah bud -t my-app .
```

Comment:
`#` or `$`? There's no need for sudo AFAIK.

Reply:
Of course there isn't, I just picked the wrong symbol :facepalm:
Thanks for spotting that!